ngraph/src/ngraph/op/convolution.cpp

```diff
@@ -72,10 +72,10 @@ void op::v1::Convolution::validate_and_infer_types()
         {
             result_shape[0] = data_batch_shape[0]; // batch size
         }
-    }
-    if (filters_shape.rank().is_static() && filters_shape.rank().get_length() > 1)
-    {
-        result_shape[1] = filters_shape[0]; // filter channel size
+        if (filters_shape.rank().is_static() && filters_shape.rank().get_length() > 1)
```

Comment:
Please add type_prop test for this changes

Reply:
I added a test